core-web/libs/edit-content/src/lib/fields/dot-edit-content-tag-field/dot-edit-content-tag-field.component.ts

```diff
@@ -23,4 +28,29 @@ import { DotSelectItemDirective } from '@dotcms/ui';
 })
 export class DotEditContentTagFieldComponent {
     @Input() field: DotCMSContentTypeField;
+
+    private readonly editContentService = inject(DotEditContentService);
+    private readonly controlContainer = inject(ControlContainer);
+
+    options!: Observable<string[]>;
```

Comment:
`options$`* We use that notation to make difference between Observables and properties/methods

Reply:
:O totally forget this!! fixed and thanks!